examples/chef/README.md

```diff
@@ -18,10 +18,13 @@ All device types available (.zap files) are found inside the `devices` folder.
 ## Building your first sample
 
 1. Make sure you have the toolchain installed for your desired target.
-2. Run `chef.py` the first time to create a `config.yaml` configuration file. If
+2. Install `zap-cli` downloading a
+   [binary release of ZAP](https://github.com/project-chip/zap/releases) and
+   adding it to your path/.
```

Comment:
zap-cli doesn't exist on some platforms, most notably ARM.  It might be better to point to our general "here is where to get ZAP" instructions, which also describe how to use a source ZAP repo...

Reply:
Makes sense, ptal